usr/src/uts/i86pc/os/intr.c

```diff
@@ -471,6 +472,7 @@
 #include <sys/hypervisor.h>
 #endif
 
+boolean_t kernel_preempt_enable = B_FALSE;
```

Comment:
How about something like `sys_rtt_kpreempt_enable`? I'm also open to suggestions.